### PR TITLE
added migration files

### PR DESCRIPTION
**Pull request by aaron-macdonald**
Two table migrations set up for 'users' and 'pings'

### CODE REVIEW

**Review by harrison-symes — CHANGES_REQUESTED** [review]
foreign key columns on pings table should be integers (not strings)

File: server/knexfile.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const path = require('path')
+
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: path.join(__dirname,'dev.sqlite3')
```
Comment (don-smith) [inline server/knexfile.js]: add space between parameters

File: server/knexfile.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    migrations: {
+      tableName: 'knex_migrations',
+      directory: path.join(__dirname,'/migrations')
```
Comment (don-smith) [inline server/knexfile.js]: add space between parameters

File: server/knexfile.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    seeds: {
+      directory: path.join(__dirname,'/seeds')
```
Comment (don-smith) [inline server/knexfile.js]: add space between parameters

File: server/knexfile.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    }
+  }
+
+};
```
Comment (don-smith) [inline server/knexfile.js]: remove semicolon

File: server/knexfile.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      user: 'username',
+      password: 'password'
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: 'knex_migrations',
+      directory: path.join(__dirname,'/migrations')
```
Comment (don-smith) [inline server/knexfile.js]: add space between parameters

File: server/knexfile.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    seeds: {
+      directory: path.join(__dirname,'/seeds')
```
Comment (don-smith) [inline server/knexfile.js]: add space between parameters

File: server/migrations/20171024102115_users.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (don-smith) [inline server/migrations/20171024102115_users.js]: remove blank line

File: server/migrations/20171024102115_users.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+exports.up = function(knex, Promise) {
```
Comment (don-smith) [inline server/migrations/20171024102115_users.js]: I thought the linter would pick up on the lack of spaces after `function`. And on the next line.

File: server/migrations/20171024103226_pings.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (don-smith) [inline server/migrations/20171024103226_pings.js]: remove blank line

File: server/migrations/20171024103226_pings.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+exports.up = function(knex, Promise) {
```
Comment (don-smith) [inline server/migrations/20171024103226_pings.js]: space after `function`

File: server/migrations/20171024103226_pings.js [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+  return knex.schema.createTableIfNotExists('pings', function(table) {
+    table.increments().primary()
+    table.integer('sender_id')
+    table.integer('recipient_id')
+    table.string('image_url')
+  })
+}
+
+exports.down = function(knex, Promise) {
```
Comment (don-smith) [inline server/migrations/20171024103226_pings.js]: space after `function`

**Review by don-smith — CHANGES_REQUESTED** [review]

**Pull request merged**